src/pages/JobProfile/SkillSetPage.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import PageLayout from '../../components/PageLayout';
+import { Box, Button, Typography, ToggleButton, useTheme } from '@mui/material';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import PropTypes from 'prop-types';
+import jobSkills from './components/mockData2';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Renders the Skill Set page.
+ * @returns {JSX.Element} Skill Set page component
+ */
+function SkillSetPage(props) {
+  const [skillList, setSkillList] = useState([]);
+  const theme = useTheme();
+
+  const handleSelectedSkill = (event, selectedSkill) => {
+    setSkillList(selectedSkill);
+  };
```

Comment:
You can select multiple buttons with on click. Please fix this

Reply:
that's  because the skills are duplicated